section-05-production-model-package/regression_model/processing/features.py

```diff
@@ -0,0 +1,52 @@
+from typing import List
+
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+
+
+class TemporalVariableTransformer(BaseEstimator, TransformerMixin):
+    # Temporal elapsed time transformer
+
+    def __init__(self, variables: List[str], reference_variable: str):
+
+        if not isinstance(variables, list):
+            raise ValueError("variables should be a list")
+
+        self.variables = variables
+        self.reference_variable = reference_variable
+
+    def fit(self, X: pd.DataFrame, y: pd.Series = None):
+        # we need this step to fit the sklearn pipeline
+        return self
+
+    def transform(self, X: pd.DataFrame) -> pd.DataFrame:
+
+        # so that we do not over-write the original dataframe
+        X = X.copy()
+
+        for feature in self.variables:
+            X[feature] = X[self.reference_variable] - X[feature]
+
+        return X
+
+
+# categorical missing value imputer
```

Comment:
this comment is not correct, this is the categorical variable mapper. Can we change or remove the comment?